skeleton/templates/cdk/src/api-stack.ts.mustache

```diff
@@ -67,6 +68,8 @@ export class ApiStack extends Stack {
                 enablePerformanceInsights: true,
             }),
             defaultDatabaseName: "api",
+            storageEncrypted: true,
+            readers: [],
```

Comment:
This shouldn't be required, as this should be the default, according to my tests.

Reply:
I just spun up a stack and without readers it created a reader instance.